.vscode/extensions.json

```diff
@@ -0,0 +1,12 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+  // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "esbenp.prettier-vscode",
+    "editorconfig.editorconfig",
+    "juanblanco.solidity",
+    "golang.go",
```

Comment:
My primary motivation for this PR was to standardize the prettier config. I included these two other extensions which I assume everyone is using for our other primary languages, but we can remove these if anyone objects.